File: ecopymobile/myapp/lib/model/clientRequestUpdate.dart
import 'dart:ffi';

import 'package:json_annotation/json_annotation.dart';

part 'clientRequestUpdate.g.dart';

@JsonSerializable()
class ClientRequestUpdate {
  bool? active;
  String? firstName;
  String? lastName;
  String? middleName;
  int? gender;
  int? cityId;
  String? address;
  DateTime? birthDate;
  String? email;
  String? username;
  String? phoneNumber;

  ClientRequestUpdate() {}

  factory ClientRequestUpdate.fromJson(Map<String, dynamic> json) =>
      _$ClientRequestUpdateFromJson(json);

  /// Connect the generated [_$ClientRequestUpdateToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$ClientRequestUpdateToJson(this);
}

File: eCopy.Model/Requests/ClientRequestUpdate.cs
using System;
using eCopy.Model.Enum;

namespace eCopy.Model.Requests
{
    public class ClientRequestUpdate
    {
        public bool? Active { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string MiddleName { get; set; }
        public Gender Gender { get; set; }
        public int CityId { get; set; }
        public string Address { get; set; }
        public DateTime BirthDate { get; set; }
        public string Email { get; set; }
        public string Username { get; set; }
        public string PhoneNumber { get; set; }

    }
}
